src/com/powem/inv/algos/ConsistentHashing.java

```diff
@@ -0,0 +1,90 @@
+package com.powem.inv.algos;
+
+import java.util.TreeMap;
```

Comment:
move the import down below the comments to make it easier to copy -paste when submitting